src/components/Router.js

```diff
@@ -75,7 +76,7 @@ export default class Router extends Component {
       ({ pathname, search }) => this.setState({ pathname, search })
     )
 
-    if (isFirecloud() && new URLSearchParams(window.location.search).has('fcredir')) {
+    if (isFirecloud() && _.has('fcredir', qs.parse(Nav.history.location.search, { ignoreQueryPrefix: true, plainObjects: true }))) {
```

Comment:
TOL: `URLSearchParams` might work just as well with `Nav.history.location` as it does with `window.location.search`... depending on how it handles the presence/absence of a query prefix. That said, it's probably better to standardize on `qs.parse` if that's what we're using. To that end, there will soon be a 3rd invocation of `qs.parse` in the code from my Fence PR, which may mean that it's time for some clean-up.

One way to clean it up is to have `Router` parse both `Nav.history.location.search` and `window.location.search` and pass both (or a union) down as props. Probably beyond the scope of this PR. Just looking for reactions to the idea here.